src/panels/lovelace/entity-rows/hui-script-entity-row.ts

```diff
@@ -64,15 +62,13 @@ class HuiScriptEntityRow extends LitElement implements EntityRow {
     `;
   }
 
-  protected renderStyle(): TemplateResult {
-    return html`
-      <style>
-        mwc-button {
-          color: var(--primary-color);
-          font-weight: 500;
-          margin-right: -0.57em;
-        }
-      </style>
+  static get styles(): CSSResult {
+    return css`
+      mwc-button {
+        color: var(--primary-color);
```

Comment:
Out of scope for this PR, but everywhere we can remove the color and font-weight style properties for `mwc-button`, it defaults to the right values.

Reply:
https://github.com/home-assistant/home-assistant-polymer/pull/2780